src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -252,11 +252,11 @@ class ClientChannelControlHelper
 // Synchronous callback from chand->resolving_lb_policy to process a resolver
 // result update.
 static bool process_resolver_result_locked(
-    void* arg, const grpc_channel_args& args, const char** lb_policy_name,
+    void* arg, grpc_core::Resolver::Result* result, const char** lb_policy_name,
```

Comment:
??? Unrelated change? Maybe the merge commit history is broken? May need some clever rebasing if possible